app/Config/CORS.php

```diff
@@ -0,0 +1,36 @@
+<?php namespace Config;
+
+use CodeIgniter\Config\BaseConfig;
+
+/**
+ * Class CORS
+ *
+ * Stores the default settings for the CORS, if you choose to use it. The
+ * values here will be read in and set as defaults for the site. If needed,
+ * they can be overridden on a page-by-page basis.
+ *
+ * Suggested reference for explanations:
+ *   - https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
+ *   - https://www.php.net/manual/en/function.header.php
+ *
+ * @author Marcelo Ratton <www.MarceloRatton.com>
+ * @package Config
+ */
+class CORS extends BaseConfig
```

Comment:
Add new config class in `app` directory is bc break without handling at `system`.

Reply:
I don't think so, cause at the default configuration is **false** so it will not be used by default

Reply:
It instantiated in `system/HTTP/Response.php`, that's why it make break, see my comment at https://github.com/codeigniter4/CodeIgniter4/pull/3706#discussion_r508553713

Reply:
ok I agree,
but this will make almost a copy of the ""Config/CORS.php"" file into **System/HTTP** folder ... or I can avoid the file in **Config** folder? just having a empty file/class there?
Cause I'm trying to keep as simple as possible

that is it?